### PR TITLE
Replaced toast with Snackbar in MultimediaEditFieldActivity

### PR DESCRIPTION
**Pull request by Giyutomioka-SS**
## Purpose / Description
Replaced toast with Snackbar in MultimediaEditFieldActivity 

## Fixes
Related to #13411

## How Has This Been Tested?
Tested on Realme 6

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration (SDK version(s), emulator or physical, etc)

## Learning (optional, can help others)
_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [ ] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/activity/MultimediaEditFieldActivity.kt [hunk 1]
```diff
@@ -80,7 +81,7 @@ class MultimediaEditFieldActivity : AnkiActivity(), OnRequestPermissionsResultCa
         val intent = this.intent
         val extras = getFieldFromIntent(intent)
         if (extras == null) {
-            UIUtils.showThemedToast(this, getString(R.string.multimedia_editor_failed), false)
+            showSnackbar(getString(R.string.multimedia_editor_failed), Snackbar.LENGTH_SHORT)
```
Comment (SanjaySargam) [inline AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/activity/MultimediaEditFieldActivity.kt:84]: The required snackbar duration is Long
So you can remove this line `Snackbar.LENGTH_SHORT`

File: AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/activity/MultimediaEditFieldActivity.kt [hunk 2]
```diff
@@ -316,7 +315,7 @@ class MultimediaEditFieldActivity : AnkiActivity(), OnRequestPermissionsResultCa
 
     private fun cancelActivityWithAssertionFailure(logMessage: String) {
         Timber.e(logMessage)
-        UIUtils.showThemedToast(this, getString(R.string.mutimedia_editor_assertion_failed), false)
+        showSnackbar(getString(R.string.mutimedia_editor_assertion_failed), Snackbar.LENGTH_SHORT)
```
Comment (SanjaySargam) [inline AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/activity/MultimediaEditFieldActivity.kt:318]: Same here also

**Review by SanjaySargam — APPROVED** [review]
LGTM!! 🎉

File: AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/activity/MultimediaEditFieldActivity.kt [hunk 3]
```diff
@@ -293,20 +294,12 @@ class MultimediaEditFieldActivity : AnkiActivity(), OnRequestPermissionsResultCa
                 recreateEditingUIUsingCachedRequest()
                 return
             }
-            UIUtils.showThemedToast(
-                this,
-                resources.getString(R.string.multimedia_editor_audio_permission_refused),
-                true
-            )
+            showSnackbar((R.string.multimedia_editor_audio_permission_refused), Snackbar.LENGTH_SHORT)
```
Comment (lukstbit) [inline AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/activity/MultimediaEditFieldActivity.kt:297]: Please remove the extra parentheses around the string id.

**Review by lukstbit — COMMENTED** [review]
One small issue.

File: AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/activity/MultimediaEditFieldActivity.kt (hunk 3 above)
Comment (Giyutomioka-SS) [reply AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/activity/MultimediaEditFieldActivity.kt]: Unit test are failing after removing those extra parentheses

**Review by lukstbit — APPROVED** [review]
Thank you!

**Review by BrayanDSO — CHANGES_REQUESTED** [review]
I'm afraid this wasn't tested. These toasts are shown when an activity is being closed, and since the snackbars are bound to an activity, they are finished together with the activity and don't show up

**Pull request closed without merging**